prototype_source/pt2e_quant_ptq_static.rst

```diff
@@ -53,6 +53,7 @@ The PyTorch 2.0 export quantization API looks like this:
 .. code:: python
 
```

Comment:
I can't comment up there by in L25 it still says "Dynamo Export." Should we change this to just "Export" since dynamo is hidden now?

Reply:
Also I noticed in L193 we're still importing `prepare_fx`, `convert_fx` etc. I guess these were incorrectly copied over from the other tutorials